### PR TITLE
Finish Login-page at least so far to be able to proceed working

### PR DESCRIPTION
**Pull request by RolfBuchwaldAlvarez**
too much work for too little outcome... 

thank you mui !

Fixes: #6

### CODE REVIEW

File: curiocityfrontend/src/pages/userLoginSimplified/UserLoginSF.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    backgroundColor: "white",
+    opacity: "60%",
+    height: "50%",
+    padding: 0,
+    margin: 0,
+  },
+  gridItem: {
+    paddingTop: "15%",
+  },
+  text1: {
```
Comment (marwinburesch) [inline curiocityfrontend/src/pages/userLoginSimplified/UserLoginSF.js:30]: I recommend renaming this class, maybe textPrimary?

File: curiocityfrontend/src/pages/userLoginOriginal/UserLogin.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+import Container from "@material-ui/core/Container";
+import Grid from "@material-ui/core/Grid";
+import makeStyles from "@material-ui/core/styles/makeStyles";
+import LogInSection from "./components/LogInSection";
+
+const useStyles = makeStyles((theme) => ({
+  container: {
+    display: "flex",
+    flexDirection: "column",
+    justifyContent: "flex-end",
```
Comment (marwinburesch) [inline curiocityfrontend/src/pages/userLoginOriginal/UserLogin.js:11]: I am assuming adding flex rules to this class is a convention you've decided on. Is that correct? I like choosing this approach compared to the MUI `<Box display="flex">` component. 👍

Comment (RolfBuchwaldAlvarez) [reply curiocityfrontend/src/pages/userLoginOriginal/UserLogin.js:11]: inspired by Caro

File: curiocityfrontend/src/pages/userLoginSimplified/UserLoginSF.js (hunk 1 above)
Comment (RolfBuchwaldAlvarez) [reply curiocityfrontend/src/pages/userLoginSimplified/UserLoginSF.js:30]: done ✌️

**Pull request merged**